### ISSUE TITLE
Nuget directive not working?

### ISSUE THREAD

**Issue by soondobu** [premake/premake-core#646]
```
project "console"
  nuget { "Google.Protobuf:3.1.0" }
  targetname "console"
  kind "ConsoleApp"
  language "C#"
  targetdir (BIN_DIR .."/%{cfg.buildcfg}")
```

This very simple nuget specification looks like it conforms to the docs, but I get the following error:

```
Building configurations...
Running action 'vs2013'...
Error: [string "src/actions/vstudio/vs2010_nuget.lua"]:32: attempt to index global 'cs2005' (a nil value)
```

Am I doing something obviously wrong here? I'm using the latest version of premake5.exe (premake5-10 alpha) with Visual Studio 2013 Update 5.

**Comment by aleksijuvani** [comment]
I think this was fixed by @tvandijck in commit 1d0a35b1d7abbfc6d96b7cf565d35cf35da569c1, so if I'm not mistaken, this should be fixed in the next version. In the meantime, you can build Premake from `master` to get these changes.

You should know the C# NuGet support still has a couple of issues that I'm hoping to iron out in the near future (see issue #569).

**Comment by samsinsane** [comment]
Based on what @aleksijuvani has said, I'm going to close this as it is resolved.
